README.md

```diff
@@ -1,12 +1,122 @@
 # percy-java-selenium
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.percy/percy-java-selenium/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.percy/percy-java-selenium)
-
+![Test](https://github.com/percy/percy-java-selenium/workflows/Test/badge.svg)
 
 [Percy](https://percy.io) visual testing for Java Selenium.
 
-## Resources
+## Installation
+
+npm install `@percy/cli`:
+
+```sh-session
+$ npm install --save-dev @percy/cli
+```
+
+Add percy-java-selenium to your project dependencies. If you're using Maven:
+
+``` xml
+<dependency>
+  <groupId>io.percy</groupId>
+  <artifactId>percy-java-selenium</artifactId>
+  <version>1.0.0</version>
+</dependency>
+```
+
+If you're using a different build system, see https://search.maven.org/artifact/io.percy/percy-java-selenium for details for your specific system.
+
+## Usage
+
+This is an example test using the `percy.snapshot` function.
+
+``` java
+// import ...
+import io.percy.selenium.Percy;
+
+public class Example {
+  private static WebDriver driver;
+  private static Percy percy;
+
+  public static void main(String[] args) {
+    FirefoxOptions options = new FirefoxOptions();
+    options.setHeadless(true);
+    driver = new FirefoxDriver(options);
+    percy = new Percy(driver);
+
+    driver.get("https://example.com");
+    percy.snapshot("Java example");
+  }
+}
+```
+
+Running the test above normally will result in the following log:
+
+```sh-session
+[percy] Percy is not running, disabling snapshots
+```
+
+When running with [`percy
+exec`](https://github.com/percy/cli/tree/master/packages/cli-exec#percy-exec), and your project's
+`PERCY_TOKEN`, a new Percy build will be created and snapshots will be uploaded to your project.
+
+```sh-session
+$ export PERCY_TOKEN=[your-project-token]
+$ percy exec -- [java test command]
+[percy] Percy has started!
+[percy] Created build #1: https://percy.io/[your-project]
+[percy] Snapshot taken "Java example"
+[percy] Stopping percy...
+[percy] Finalized build #1: https://percy.io/[your-project]
+[percy] Done!
+```
+
+## Configuration
+
+`percy.snapshot(driver, name, [widths], minHeight, enableJavaScript, percyCSS)`
```

Comment:
I think you want `widths[]` here. Unless you're trying to denote optional parameters, in which case you'll want to include the previous comma in the brackets and repeat for the remaining options